ui/src/components/home/ResourceTab.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Typography } from '@mui/material';
+import { FC, ReactElement } from 'react';
+import TextSnippetIcon from '@mui/icons-material/TextSnippet';
+
+const GroupTab: FC = (): ReactElement => {
+  return (
+    <Box
+      sx={{
+        padding: '1em',
+        border: '1px solid #ccc',
```

Comment:
why do only resource and song tab have border ya? is the design like that? cuz it seems inconsistent to me..

side note: think in the future we can incld the border to the theme, p sure we r gonna use this style in other components too lol